### ISSUE TITLE
Some gamepad tests are mysteriously failing after Bevy 0.15 upgrade

### ISSUE THREAD

**Issue by alice-i-cecile** [Leafwing-Studios/leafwing-input-manager#665]
> Definitely share your concern: it's a blocking issue. I want to split it out though to be sure we can properly review and discuss the fix.

_Originally posted by @alice-i-cecile in https://github.com/Leafwing-Studios/leafwing-input-manager/issues/664#issuecomment-2529419477_
            
We should figure out how severe this is (does this actually happen for real users), then explore a fix. If this is real, we can't ship the new version until it's fixed.

**Comment by mrkiffie** [comment]
Looks like this might have been resolved in #666?
